tox.ini

```diff
@@ -153,3 +153,9 @@ basepython = python3.9
 extras =
 deps =
 commands = python -c 'import plasmapy'
+
+[testenv:mypy]
+deps =
+    mypy >= 1.8.0
+commands =
+    mypy plasmapy --install-types --non-interactive --show-error-context --show-error-code-links --pretty
```

Comment:
We'll probably want to play with these flags to get the error reporting settings that work best for us.